pgcli/completion_refresher.py

```diff
@@ -68,6 +70,13 @@ def _bg_refresh(self, pgexecute, special, callbacks):
             # break statement.
             continue
 
+        # Load history into pgcompleter so it can learn user preferences
+        n_max_recent = 100
+        n_recent = history and min(n_max_recent, len(history))
+        if n_recent:
+            for recent in history[-n_recent:]:
```

Comment:
This could be rewritten as:

```
n_recent = history and 100
for recent in history[-n_recent:]:
```

If you ask for last 100 items and history is only 10 items, python will give you only the 10 items. No need to be cautious about array out of bounds. 

Proof: 

```
>>> a = [1,2,3]
>>> a[-5:]
[1, 2, 3]
>>> a[-2:]
[2, 3]
>>> a[-1:]
[3]
```